neurots/generate/diametrizer.py

```diff
@@ -355,6 +355,24 @@ def diametrize_constant_per_neurite(neuron, neurite_type=None):
             sec.diameters = mean_diam * np.ones(len(sec.diameters))
 
 
+def diametrize_uniform(neuron, neurite_type=None, *, diam_params):
+    """Set all diameters of a morphio-neuron to a given value.
+
+    Args:
+        neuron (morphio.mut.Morphology): The morphology that will be diametrized.
+        neurite_type (morphio.SectionType): Only the neurites of this type are diametrized.
+        diam_params (dict): The model parameters (should only contain a 'diameter' entry).
+    """
+    diameter = diam_params.get(neurite_type.name, None)
+    if diameter is None:
```

Comment:
can you instead put a DEFAULT_DIAMETER=1 ?

Reply:
Sure, done

Reply:
Ah wait, I think I misunderstood. Do you mean instead of doing nothing when it's `None`?

Reply:
yes, no? I'm not sure either what it does if you hit this `return`

Reply:
ah ok ok, all good then!